src/command.ts

```diff
@@ -76,6 +83,8 @@ const CommandDescriptionMap: Record<string, string> = {
     'Encontra as localizações que tem você nos registros.',
   [PortugueseCommandAlias.ListCountryMemberCount]:
     'Lista a quantidade de membros por países.',
+  [PortugueseCommandAlias.RankCountryMemberCount]:
+    'Classifica os países baseado no número de membros',
```

Comment:
I think here we can use some neutral Portuguese, as "membros" assume male gender.

```suggestion
    'Classifica os países baseado no número de pessoas registradas.',
```